package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "malevich",
+  "version": "1.0.0",
+  "description": "TypeScript / React / HTML5 Training Exercise",
+  "main": "./dist/bundle.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
npm supports pre/post hooks. it'd be nice to use this to autoinstall the typings when you run `npm install`. see https://docs.npmjs.com/misc/scripts